src/components/Banner/BaseAnnouncementBanner/index.tsx

```diff
@@ -60,14 +60,15 @@ export default function BaseWalletBanner() {
       </ThemedText.HeadlineMedium>
 
       <ButtonRow>
-        {isIOS ? (
+        {isIOS || (isAndroidGALaunched && isAndroid) ? (
```

Comment:
variable naming seems inconsistent, sometimes it's `isAndroidGALaunched` and sometimes it's just `isAndroidLaunched` - i thought these were referring to two different things tbh, maybe just consolidate to `isAndroidGALaunched`?